src/Domain/CoordsToDifferentFrame.hpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.Add commentMore actions
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+namespace ylm {
+template <typename Frame>
+class Strahlkorper;
+}  // namespace ylm
+template <size_t Dim>
+class Domain;
+namespace domain::FunctionsOfTime {
+class FunctionOfTime;
+}  // namespace domain::FunctionsOfTime
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+// Transforms cartesian coordinates from one frame to another, by calling
+// block_logical_coordinates and calling the correct map functions.
```

Comment:
Make this a proper Doxygen comment, using `/*!` and `*/`, and put in some more docs. For example, put in the possible values for `SrcFrame` and `DestFrame`.